Bug: missing NPM `packages/lexical-playground/package.json`'s `devDependencies`: `@rollup/plugin-babel @babel/plugin-transform-flow-strip-types @babel/preset-react`

Hi, I have a project with extracted playground into a separate repo for tag v0.16.0:

When doing `npm run --prefix packages/lexical-playground build-prod`, I'm getting an error `Error [ERR_MODULE_NOT_FOUND]: Cannot find package '@rollup/plugin-babel'`

Then by adding it to the prior `npm install` run, I'm getting a similar errors for `@babel/plugin-transform-flow-strip-types @babel/preset-react`

Adding all three to `npm install` makes it work:
```shell
npm install --force --prefix packages/lexical-playground @rollup/plugin-babel @babel/plugin-transform-flow-strip-types @babel/preset-react
npm run --prefix packages/lexical-playground build-prod
```

So I assume this is indeed only a matter of adding these to `devDependencies`?

Removing `--force` produces this error:
```
npm error code ERESOLVE
npm error ERESOLVE unable to resolve dependency tree
npm error
npm error While resolving: lexical-playground@0.16.0
npm error Found: vite@5.2.[13](https://github.com/vadimkantorov/lexical-playground-only/actions/runs/9437767079/job/25994006725#step:4:14)
npm error node_modules/vite
npm error   dev vite@"^5.2.11" from the root project
npm error
npm error Could not resolve dependency:
npm error peer vite@"^2" from vite-plugin-replace@0.1.1
npm error node_modules/vite-plugin-replace
npm error   dev vite-plugin-replace@"^0.1.1" from the root project
npm error
npm error Fix the upstream dependency conflict, or retry
npm error this command with --force or --legacy-peer-deps
npm error to accept an incorrect (and potentially broken) dependency resolution.
```

--- a/packages/lexical-devtools/package.json
+++ b/packages/lexical-devtools/package.json
@@ -39,6 +39,8 @@
     "zustand": "^4.5.1"
   },
   "devDependencies": {
+    "@babel/plugin-transform-flow-strip-types": "^7.24.7",
+    "@babel/preset-react": "^7.24.7",
     "@lexical/devtools-core": "0.16.0",
     "@rollup/plugin-babel": "^6.0.4",
     "@types/react": "^18.2.46",

--- a/packages/lexical-playground/package.json
+++ b/packages/lexical-playground/package.json
@@ -37,6 +37,9 @@
     "yjs": ">=13.5.42"
   },
   "devDependencies": {
+    "@babel/plugin-transform-flow-strip-types": "^7.24.7",
+    "@babel/preset-react": "^7.24.7",
+    "@rollup/plugin-babel": "^6.0.4",
     "@rollup/plugin-commonjs": "^25.0.7",
     "@types/lodash-es": "^4.14.182",
     "@vitejs/plugin-react": "^4.2.1",

